### ISSUE TITLE
[React Native Gallery -> Permissions]: The Name property of an element should not contain names like "Microsoft.*.*' or 'Windows.*.*' as these are not usually informative. 

### ISSUE THREAD

**Issue by AgneLukoseviciute** [microsoft/react-native-gallery#314]
**Repro-Steps:**
Launch the React Native Gallery app.
Navigate to ‘Permissions’ button using tab key, invoke it.
Now run fastpass and observe. 

**Actual Result:**
The Name property of an element should contain names like "Microsoft.*.*' or 'Windows.*.*' as these are not usually informative. 
 
**Expected Result:**
The Name property of an element should not include the elements class name (such as Microsoft.*.*' or 'Windows.*.*).

**User impact:**
Users with visual impairments who rely on screen reader will not be able to know the name and purpose of the button if they is not proper name is defined. 

Internal bug link: https://microsoft.visualstudio.com/DefaultCollection/OS/_workitems/edit/43367878

**Comment by AgneLukoseviciute** [comment]
Are these namespace names or react-native-permissions properties? in which case we can probably argue with the a11y team that it makes sense to leave those names and get that bug closed out. 
![image](https://user-images.githubusercontent.com/41223743/221709110-76e30fd4-5d15-4617-88f8-c1d8e5232804.png)

**Comment by chrisglein** [comment]
Agreed, the accessibility names here match what is displayed in the interface, and contextually what's in the interface makes sense given what the module is doing. This one is worth pushing back on. Will do when we do a pass on the block of sev3s.
